### ISSUE TITLE
In M1 Mac, there is a error when I am running 'make' command. How do I solve this problem? 

### ISSUE THREAD

**Issue by Yin-bo-de** [libevent/libevent#1390]
the error infomations is follow:

/bin/sh ../libtool  --tag=CC   --mode=link gcc -I.. -I../compat -Wall -fno-strict-aliasing -D_THREAD_SAFE   -o test-time test-time.o ../libevent_core.la -lresolv 
libtool: link: gcc -I.. -I../compat -Wall -fno-strict-aliasing -D_THREAD_SAFE -o .libs/test-time test-time.o  ../.libs/libevent_core.dylib -lresolv
gcc -DHAVE_CONFIG_H -I. -I..    -I.. -I../compat -Wall -fno-strict-aliasing -D_THREAD_SAFE -c -o regress.o regress.c
regress.c:65:10: fatal error: 'regress.gen.h' file not found
#include "regress.gen.h"
         ^~~~~~~~~~~~~~~
1 error generated.
make[3]: *** [regress.o] Error 1
make[2]: *** [all] Error 2
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2

**Comment by azat** [comment]
This file is generated with python.
Maybe you don't have python interpreter installed?

**Comment by Yin-bo-de** [comment]
> 

No, I have already installed python interpreter. is there any way to locate this problem, please?

**Comment by azat** [comment]
Sorry for the delay, could you please try build with cmake and post the full logs.
